server/controllers/article.js

```diff
@@ -95,7 +102,7 @@ class Article {
     const page = parseInt(request.query.page, 10) || 1;
 
     const articles = await articleRepo.getArticles(limit, page);
-    if (!articles) {
+    if (articles.count === 0) {
```

Comment:
`! articles.count ` ?

Reply:
@seyi-adeleke I don't understand your question on this.